tests/torch_datasets/test_pvnet_uk_regional.py

```diff
@@ -3,7 +3,9 @@
 
 from ocf_data_sampler.torch_datasets.pvnet_uk_regional import PVNetUKRegionalDataset
 from ocf_data_sampler.config import load_yaml_configuration, save_yaml_configuration
-from ocf_datapipes.batch import BatchKey, NWPBatchKey
+from ocf_data_sampler.numpy_batch.nwp import NWPBatchKey
+from ocf_data_sampler.numpy_batch.gsp import GSPBatchKey
+from ocf_data_sampler.numpy_batch.satellite import SatelliteBatchKey
```

Comment:
we can import those into numpy_batch __init__ for neater imports? I'm happy to do that if you want